docs/automatic-redirects.rst

```diff
@@ -4,29 +4,44 @@ Automatic Redirects
 Read the Docs supports redirecting certain URLs automatically.
 This is an overview of the set of redirects that are fully supported and will work into the future.
 
+Redirecting to a Page
+---------------------
+
+You can link to a specific page and have it redirect to your default version.
+This is done with the ``/page/`` URL.
+You can reach this page by going to https://docs.readthedocs.io/page/automatic-redirects.html.
+
+This allows you to create links that are always up to date.
+
+Another way to handle this is the *latest* version.
+You can set your ``latest`` version to a specific version and just always link to latest.
```

Comment:
We use 3 way of formatting for "latest" here, and could consolidate for clarity

```suggestion
Another way to handle this is the ``latest`` version.
You can set your ``latest`` version to a specific version and just always link to ``latest``.
```

Or all bold, either way works